k8s/schedulers/scheduling.go

```diff
@@ -234,8 +231,8 @@ func (vs *VineyardScheduling) Less(pod1, pod2 *framework.PodInfo) bool {
 }
 
 // PreFilter for a pod
-func (vs *VineyardScheduling) PreFilter(ctx context.Context, state *framework.CycleState, pod *v1.Pod) *framework.Status {
-	return framework.NewStatus(framework.Success, "")
+func (vs *VineyardScheduling) PreFilter(ctx context.Context, state *framework.CycleState, pod *v1.Pod) (*framework.PreFilterResult, *framework.Status) {
+	return nil, framework.NewStatus(framework.Success, "")
```

Comment:
If you decide to delete the `PreFilterPlugin`, you can delete this function directly.

Reply:
Ummm, I'm on the way to refactoring the scheduler, and we need to talk about how the scheduler works.